### ISSUE TITLE
Document PR #85968 (Fix bug in apiserver service cidr split)

### ISSUE THREAD

**Issue by MorrisLaw** [contributing-to-kubernetes/cool-kubernetes#19]
**PR:**
https://github.com/kubernetes/kubernetes/pull/85968

**ISSUE:**
bug

**SIG LABELS:**
sig/api-machinery
sig/network

**Comment by alejandrox1** [comment]
@MorrisLaw i just created the bracnh `issue-19` on this repo for us to work on this :smile:

**Comment by alejandrox1** [comment]
Please feel free to push commits to that branch and then later on we can clean the history before we merge it to master. wdyt?

**Comment by alejandrox1** [comment]
Im gonna "squash and merge" my PRs in order to avoid any "merged" commit messages on that branch.
Should make it easier to then clean up whenever we want to merge to master.

**Comment by alejandrox1** [comment]
Summary of what I just did, please criticize and suggest alternatives
**For every issue we propose we create a branch `issue-X` on this repo (upstream repo)**.

Then, while we work on the issue (the draft), we make PRs against the `issue-X` branch for the specific issue.
When you are done working on your PR (it may take multiple PRs from us to get the `issue-X` branch ready to merge to master) you merge it as **squash and merge** (to avoid any merge commit messges).

When `issue-X` branch is ready to be merged into `master` we can then cleanup and merge it by making a PR from `issue-X` to `master`.
After that, we share the work and look for feedback :rocket:

**Comment by MorrisLaw** [comment]
Yeah I think this is a good idea! Thank you @alejandrox1! 

So... just to summarize:

- Create a branch, named `issue-X`, on the upstream repo
- Make iterative changes to the branch by making PRs against that branch
  - When a PR is done being reviewed and worked on, merge it as **squash and merge** into the `issue-X` branch (to avoid any merge commit messages)
- Once `issue-X` is ready to be merged, clean it up and merge the final result into `master`

**Comment by alejandrox1** [comment]
Yas (we may want to commit these instructions in the repo as well, after we try them out to see if they make sense)

**Comment by alejandrox1** [comment]
@MorrisLaw I pushed what i think is a final draft for this issue in a recent PR, please review at your convenience. Then we can see if its good to go to be published"
